Bottom: 6562aca2d162c0ad35853f8c4843fc5d90f988b3
Top:    ebef106b196b9ca590348afca8428a82520ed5a6
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 17:22:17 +0800

Refresh of debug.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index 0f3711d..81ba3ca 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -225,7 +225,6 @@ dont_probe:
 
 	/* Root offset of multi-mount */
 	len = strlen(root);
-	error(LOGOPT_ANY, "len = %d root %s name %s root[len-1] %c", len, root, name, root[len-1]);
 	if (root[len - 1] == '/') {
 		len = snprintf(fullpath, len, "%s", root);
 	} else if (*name == '/') {
diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index edbfeb0..c1fc528 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -838,8 +838,6 @@ update_offset_entry(struct autofs_point *ap, const char *name,
 	} else
 		strcpy(m_mapent, loc);
 
-	error(LOGOPT_ANY, "name %s m_key %s m_mapent %s", name, m_key, m_mapent);
-
 	ret = cache_update_offset(mc, name, m_key, m_mapent, age);
 	if (ret == CHE_DUPLICATE)
 		warn(ap->logopt, MODPREFIX
